t/24-worker-overall.t

```diff
@@ -142,6 +142,26 @@ subtest 'capabilities' => sub {
     # clear cached capabilities
     delete $worker->{_caps};
 
+    subtest 'worker instance contains PARALLEL_ONE_HOST_ONLY' => sub {
+	my $worker2 = OpenQA::Worker->new({instance => 2, apikey => 'foo', apisecret => 'bar', verbose => 1, 'no-cleanup' => 1});
+        my $capabilities2 = $worker2->capabilities;
+	# ignore keys which are not always present and also not strictly required anyways
+	delete $capabilities->{cpu_flags};
+	delete $capabilities->{cpu_opmode};
+	delete $capabilities->{cpu_modelname};
+
+	is_deeply(
+	    [sort keys %$capabilities],
+	    [
+	     qw(
+		 cpu_arch host instance isotovideo_interface_version
+		 mem_max websocket_api_version worker_class parallel_one_host_only
```

Comment:
This needs to be in alphabetical order. But to not repeat the whole previous subtest this test should probably just check `is $capabilities->{parallel_one_host_only}, 1, …`.